python/mxnet/base.py

```diff
@@ -735,7 +739,121 @@ def write_all_str(module_file, module_all_list):
 ctypes.pythonapi.PyCapsule_New.restype = ctypes.py_object
 ctypes.pythonapi.PyCapsule_GetPointer.restype = ctypes.c_void_p
 
+
 from .runtime import Features
 if Features().is_enabled("TVM_OP"):
     _LIB_TVM_OP = libinfo.find_lib_path("libtvmop")
     check_call(_LIB.MXLoadTVMOp(c_str(_LIB_TVM_OP[0])))
+
+
+def _sanity_check_params(func_name, unsupported_params, param_dict):
```

Comment:
docstring missing

Reply:
Actually this function no longer needed, removing.